clients/unshaded/src/main/java/tachyon/client/lineage/AbstractLineageClient.java

```diff
@@ -49,10 +51,10 @@ public long createLineage(List<TachyonURI> inputFiles, List<TachyonURI> outputFi
           throws FileDoesNotExistException, TachyonException, IOException {
     // TODO(yupeng): relax this to support other type of jobs
     Preconditions.checkState(job instanceof CommandLineJob, "only command line job supported");
-
     LineageMasterClient masterClient = mContext.acquireMasterClient();
     try {
-      long lineageId = masterClient.createLineage(inputFiles, outputFiles, (CommandLineJob) job);
+      long lineageId = masterClient.createLineage(stripURIList(inputFiles),
```

Comment:
Thanks for the comment. Actually, that was what I was initially doing since LineageMasterClinet#createLineage was internally converting uris to strings already (although through toString() rather than getPath()). However, I went with this solution to keep it symmetric to what is done in AbstractTachyonFileSystem#create(TachyonURI, CreateOptions).
Feel free to lemme know what you think its best.